client/lib/like-store/like-store.js

```diff
@@ -172,7 +169,7 @@ LikeStore = {
 	},
 };
 
-if ( config( 'env' ) === 'development' ) {
```

Comment:
why are all of these swapping to test?

Reply:
they aren't for development but rather for tests only. that is, these are used by the tests to reset state and the tests were failing because these were missing.

the `test.json` config file sets the `env` as `development` (not sure why)

@see d89d088

Reply:
oh, i see. cool 👍